contents/product-engineers/mobile-app-metrics.md

```diff
@@ -0,0 +1,579 @@
+---
+date: 2024-09-16
+title: 25 best mobile app metrics to track
+author:
+  - lior-neu-ner
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product metrics
+crosspost:
+  - Blog
+---
+
+## Sign up and onboarding metrics
+
+### 1. App downloads
+
+**What it is:** The number of times your app has been downloaded.
+
+**Why it's useful:** Provides a baseline metric for user acquisition and helps measure the effectiveness of marketing campaigns.
+
+**Questions to ask:**
+- How does the number of downloads vary across different platforms (iOS, Android, etc.)?
+- Is there a correlation between the number of downloads and user engagement or retention?
+- How do different marketing channels impact the number of downloads?
+
+**Insight type:** [Trend](/docs/product-analytics/trends/overview)
```

Comment:
```suggestion
**How to track it:** Send an event to PostHog each time your app is downloaded and track it using a [trend](/docs/product-analytics/trends/overview). Breakdown by `Device Type` or `OS` to see the distribution by platform.
```

I think each one should be more like this – feel free to change this if it isn't right, I just wanted to have an example. It doesn't need to super detailed, but just saying use a trend insight doesn't feel super satisfying.